### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by mattinannt** [formbricks/formbricks#4505] [Announcements]
We're thrilled to announce the release of Formbricks 3.0, packed with powerful new features and improvements that make gathering insights and creating exceptional experiences easier than ever. 🎉🎉🎉

---

> [!IMPORTANT]    
> **Don't upgrade to 3.0 if you need SSO, user identification or cluster support on the community edition!**
With Formbricks 3.0, we're making some strategic changes to ensure long-term sustainability while keeping our core commitment to open source. While the Community Edition [got more powerful](https://formbricks.com/blog/formbricks-3-0), we're moving [some advanced features](https://formbricks.com/docs/self-hosting/license) to the Enterprise Edition.
If you upgrade to 3.0 and run the data migration, there IS NO WAY BACK to 2.7.2 - if you need to use SSO or one of the other previously free features, either stick with 2.7.x or [reach out](mailto:hola@formbricks.com) for a custom quote.

---

# Major new features ✨
With our small team, we keep working hard to make Formbricks better for everyone: Our community as well as our Enterprise customers. Here are the highlights of Formbricks 3.0:

- **Multi-language support**: 🌍 Complete UI now available in French, German, and Portuguese. Need another language? [Get in touch 👋](mailto:hola@formbricks.com)
- **Email Follow-ups**: 📧 Automatically send custom follow-up emails to your respondents, team mates or yourself
- **New "Contact" question type**: 📝 Capture and analyze contact information from your surveys.
- **AI Insights**: 🤖 Public beta for AI Insights generation for free text responses (BYO model on self-hosted)
- **Formbricks Wordpress Plugin**: 🔌 Embed surveys in your WordPress site with ease, [learn more here.](https://formbricks.com/blog/wordpress-surveys-plugin)
- **Teams and Access Rights**: 👥 Manage your team and access rights to specific surveys
- **Formbricks Cluster support & Helm Chart**: ⚓ Deploy Formbricks on Kubernetes
- **Advanced Logic**: 🧮 Add more complex logic to your surveys
- **Variables**: 🔢 Add text and numeric variables to keep track of scores, prices, etc.
- **Performance**: ⚡ Decrease for in-product survey server load by ~80%
- **Automatic Migrations**:✨ Makes upgrading to future versions of Formbricks a breeze.
- Better filters, templates, and tons of bug fixes.

Learn more in our [blog post](https://formbricks.com/blog/formbricks-3-0) for this release.

For details about the migration to 3.0.0 please visit our [migration guide](https://formbricks.com/docs/self-hosting/migration-guide).

Thanks to all the contributors who help make Formbricks more robust and performant. 💪🙏

---

## What's Changed
* feat: product model revamp by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4353
* feat: contacts revamp by @pandeymangg in https://github.com/formbricks/formbricks/pull/3399
* feat: new data migrations approach by @pandeymangg in https://github.com/formbricks/formbricks/pull/4466
* feat: survey follow ups by @pandeymangg in https://github.com/formbricks/formbricks/pull/4247
* feat: team users clean up by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4448
* feat: french translations by @Dhruwang in https://github.com/formbricks/formbricks/pull/4395
* feat: add multi arch to data-migration by @philippdormann in https://github.com/formbricks/formbricks/pull/4421
* feat: add intercom by @jobenjada in https://github.com/formbricks/formbricks/pull/4363
* feat: adds vercel style guide in surveys package WIP by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4401
* feat: allowed assigning of variable to open text number question by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4334
* feat: Auth module revamp by @Dhruwang in https://github.com/formbricks/formbricks/pull/4335
* feat: character limitations to free text question by @DeepaPrasanna in https://github.com/formbricks/formbricks/pull/4047
* feat: created icon bar component in survey overview by @aryamantodkar in https://github.com/formbricks/formbricks/pull/4240
* feat: logic fallback option added by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4306
* feat: updated org limits data migration by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4329
* feat: refactor remove branding by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4393
* fix: add time to exports by @Dhruwang in https://github.com/formbricks/formbricks/pull/4430
* fix: adds example CSV and other fixes on the contacts page by @pandeymangg in https://github.com/formbricks/formbricks/pull/4493
* fix: auto zoom on ios by @Dhruwang in https://github.com/formbricks/formbricks/pull/4502
* fix: broken invite flow by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4387
* fix: buggy docs navigation by @Dhruwang in https://github.com/formbricks/formbricks/pull/4441
* fix: build warnings by @Dhruwang in https://github.com/formbricks/formbricks/pull/4407
* fix: button descendant console error by @Dhruwang in https://github.com/formbricks/formbricks/pull/4468
* fix: created at to integrations by @Dhruwang in https://github.com/formbricks/formbricks/pull/4343
* fix: csp header by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4384
* fix: delete own account deletes org (#4419) by @melvinjariwala in https://github.com/formbricks/formbricks/pull/4422
* fix: empty config projects by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4391
* fix: ending card url by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4364
* fix: error on modal opening by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4503
* fix: i18n issues in date picker by @Dhruwang in https://github.com/formbricks/formbricks/pull/4469
* fix: improved AI insights generation prompt by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4330
* fix: increase share rate limit by @Dhruwang in https://github.com/formbricks/formbricks/pull/4345
* fix: keyboard overlaps the survey by @Dhruwang in https://github.com/formbricks/formbricks/pull/4386
* fix: license cache issue by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4337
* fix: license checks in server actions by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4274
* fix: light mode  by @Dhruwang in https://github.com/formbricks/formbricks/pull/4423
* fix: logic jump by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4501
* fix: migrations speed by @pandeymangg in https://github.com/formbricks/formbricks/pull/4492
* fix: missing impression count by @Dhruwang in https://github.com/formbricks/formbricks/pull/4397
* fix: name regex by @Dhruwang in https://github.com/formbricks/formbricks/pull/4328
* fix: next button label translation  by @Dhruwang in https://github.com/formbricks/formbricks/pull/4398
* fix: next js 15 stale experimental flags by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4350
* fix: no displays for multi language surveys by @pandeymangg in https://github.com/formbricks/formbricks/pull/4482
* fix: onboarding invite by @Dhruwang in https://github.com/formbricks/formbricks/pull/4426
* fix: only one jump to question is allowed by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4332
* fix: ranking ques performance by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4301
* fix: react native sync by @pandeymangg in https://github.com/formbricks/formbricks/pull/4405
* fix: recall in slack integration by @Dhruwang in https://github.com/formbricks/formbricks/pull/4304
* fix: remove cache from identify endpoint to fix issues with recontact time by @mattinannt in https://github.com/formbricks/formbricks/pull/4453
* fix: removes environment id in local storage by @khAja13 in https://github.com/formbricks/formbricks/pull/4347
* fix: resolve BUG #4447 by @Surajsuthar in https://github.com/formbricks/formbricks/pull/4458
* fix: response card tags not updating by @Dhruwang in https://github.com/formbricks/formbricks/pull/4424
* fix: search on contacts page by @pandeymangg in https://github.com/formbricks/formbricks/pull/4484
* fix: summary calculation by @Dhruwang in https://github.com/formbricks/formbricks/pull/4500
* fix: survey auto complete by @Dhruwang in https://github.com/formbricks/formbricks/pull/4442
* fix: survey release and close bug for first date of the month by @SaiSawant1 in https://github.com/formbricks/formbricks/pull/4311
* fix: survey saving with styles by @Dhruwang in https://github.com/formbricks/formbricks/pull/4404
* fix: sync endpoint cache by @pandeymangg in https://github.com/formbricks/formbricks/pull/4394
* fix: translations and onboarding bug by @Dhruwang in https://github.com/formbricks/formbricks/pull/4370
* fix: typo in pricing table by @Dhruwang in https://github.com/formbricks/formbricks/pull/4377
* fix: usability improvements survey editor by @jobenjada in https://github.com/formbricks/formbricks/pull/4362
* fix: user(owner) delete bug by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4388
* fix: variable docs by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4338
* fix: vercel build by @Dhruwang in https://github.com/formbricks/formbricks/pull/4351
* fix: video embed url by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4485
* fix: webhook deleteion and updation working by @Dhruwang in https://github.com/formbricks/formbricks/pull/4431
* fix: welcome card description not updating by @Dhruwang in https://github.com/formbricks/formbricks/pull/4440
* fix: wrong invite message by @Dhruwang in https://github.com/formbricks/formbricks/pull/4470
* chore(deps): bump nanoid from 5.0.7 to 5.0.9 in the npm_and_yarn group across 1 directory by @dependabot in https://github.com/formbricks/formbricks/pull/4428
* chore: Billing to new module structure by @Dhruwang in https://github.com/formbricks/formbricks/pull/4308
* chore: Freeze client api + api docs by @Dhruwang in https://github.com/formbricks/formbricks/pull/4373
* chore: add github issue task template by @mattinannt in https://github.com/formbricks/formbricks/pull/4340
* chore: added shadcn button by @Dhruwang in https://github.com/formbricks/formbricks/pull/4392
* chore: make redis an enterprise feature by @Dhruwang in https://github.com/formbricks/formbricks/pull/4314
* chore: move ee services to new module structure by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4317
* chore: move ui components to modules by @Dhruwang in https://github.com/formbricks/formbricks/pull/4342
* chore: moves survey follow ups to community by @pandeymangg in https://github.com/formbricks/formbricks/pull/4476
* chore: next and react version bump by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4459
* chore: next version upgrade by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4291
* chore: onboarding cleanup by @Dhruwang in https://github.com/formbricks/formbricks/pull/4479
* chore: preparations for 3.0 release by @mattinannt in https://github.com/formbricks/formbricks/pull/4499
* chore: removed experience page by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4320
* chore: update all npm dependencies by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/4486
* chore: update client api rate limits by @mattinannt in https://github.com/formbricks/formbricks/pull/4425
* chore: update github issue template by @mattinannt in https://github.com/formbricks/formbricks/pull/4339
* chore: upgrade node to 22, upgrade pnpm, upgrade turbo by @mattinannt in https://github.com/formbricks/formbricks/pull/4374
* chore: upselling UI by @Dhruwang in https://github.com/formbricks/formbricks/pull/4491
* chore: vercel style guide (part 1) by @Dhruwang in https://github.com/formbricks/formbricks/pull/4451
* docs: Update license docs by @jobenjada in https://github.com/formbricks/formbricks/pull/4488
* docs: management api endpoints for contacts by @Dhruwang in https://github.com/formbricks/formbricks/pull/4494
* docs: mention `ENCRYPTION_KEY` length limit by @salim-b in https://github.com/formbricks/formbricks/pull/4360
* docs: move community from Discord to Github Discussions by @mattinannt in https://github.com/formbricks/formbricks/pull/4365
* docs: redis docs by @Dhruwang in https://github.com/formbricks/formbricks/pull/4316
* docs: update docs by @jobenjada in https://github.com/formbricks/formbricks/pull/4449

## New Contributors
* @aryamantodkar made their first contribution in https://github.com/formbricks/formbricks/pull/4240
* @DeepaPrasanna made their first contribution in https://github.com/formbricks/formbricks/pull/4047
* @Surajsuthar made their first contribution in https://github.com/formbricks/formbricks/pull/4458
* @melvinjariwala made their first contribution in https://github.com/formbricks/formbricks/pull/4422
* @philippdormann made their first contribution in https://github.com/formbricks/formbricks/pull/4421

**Full Changelog**: https://github.com/formbricks/formbricks/compare/v2.7.1...v3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/formbricks/formbricks/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by siegenthalerroger** [comment]
Seriously, helm charts are an "enterprise" feature? I may disagree with SSO (a security feature) being enterprise but at least that's kind of expected tbh... but a deployment method? Just unnecessary

**Reply by jobenjada** [reply]
hey! This is actually not correct in the release note, the Helm chart is AGPLv3, just the cluster support isn't.

I'm not 100% sure if the Helm chart is ready to be used already, I'll ping @mattinannt to follow up on this  🤓

**Reply by siegenthalerroger** [reply]
Ah phew, that's much better and more in line with what I'd like to see. Love what you guys are doing.

**Comment by flygav1** [comment]
Not being able to remove branding on self hosted version makes me very sad. can you possibly do a donation to remove branding instead of paying a monthly fee?

**Reply by bogxi** [reply]
I second. I'll happily pay a small fee to remove branding. I'm sticking to `v2.7.x` for now.

**Reply by siegenthalerroger** [reply]
Hard disagree IMO, branding is one of the least important things to be removed unless you're profiting from it (in which case you should pay something for it). Whether the pricing structure is acceptable is a different thing IMO.

**Reply by jobenjada** [reply]
@jorge-ui and @flygav1 pls reach out to hola@formbricks.com - we're planning to offer Branding Removal for a small fee to get more insight into who is using Formbricks on premise. It's not yet ready to be released publicly, but happy to provide you with a payment link via email.

**Reply by bogxi** [reply]
@jobenjada Yaaassssss! 🎄🔥

Thank you!

💌 ✅

**Reply by thenitai** [reply]
> @jorge-ui and @flygav1 pls reach out to [hola@formbricks.com](mailto:hola@formbricks.com) - we're planning to offer Branding Removal for a small fee to get more insight into who is using Formbricks on premise. It's not yet ready to be released publicly, but happy to provide you with a payment link via email.

Is this happening or should we just stick with 2.7.2?
